spec/features/users/user_can_create_an_account_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+describe 'When a user visits `/users/new`' do
+  describe 'and they enter all required params' do
```

Comment:
we could also make these contexts rather than describes, but either way, i like the robust sad path testing 👍